mmdet/models/dense_heads/ssd_head.py

```diff
@@ -95,6 +105,75 @@ def __init__(self,
             self.sampler = build_sampler(sampler_cfg, context=self)
         self.fp16_enabled = False
 
+    def _init_layers(self):
+        """Initialize layers of the head."""
+        self.cls_convs = nn.ModuleList()
+        self.reg_convs = nn.ModuleList()
+        conv = DepthwiseSeparableConvModule \
```

Comment:
May add a TODO here, indicating that we will use the registry to choose either DepthwseSeparableConvModule or ConvModule in the future. Thus, we will have a chance to reduce the option `use_depthwise` in the future.